client/packages.go

```diff
@@ -112,6 +114,42 @@ func (client *Client) List(names []string) ([]*Snap, error) {
 	return result, nil
 }
 
+func setPrice(snap *Snap, currency string) {
```

Comment:
Note that on the phone, the "price" field is deprecated, and is not used. However, as steven says, the suggested currency is always used. But this is only the case because there's not been enough developer resource to make the store app allow the user to select their preferred currency.